chemicaltoolbox/rdkit/sdf_to_tab.xml

```diff
@@ -0,0 +1,104 @@
+<tool id="sdf_to_tab" name="Extract values from an SD-file" version="@VERSION@">
+    <macros>
+        <token name="@VERSION@">2019.03.1</token>
+    </macros>
+    <description>into a tabular file using RDKit</description>
+    <requirements>
+        <requirement type="package" version="@VERSION@">rdkit</requirement>        
+    </requirements>
+    <command detect_errors="exit_code">
+<![CDATA[
+        python '$__tool_directory__/sdf_to_tab.py'
+            -i '${infile}'
+            -o '${outfile}'
+            -p '$props'
+            $header
+]]>
+    </command>
+    <inputs>
+        <param name="infile" format="sdf" type="data" label="Input SD-file"/>
+        <param name="header" type="boolean" label="Include the property name as header"
```

Comment:
Mh, interesting is there a use-case where you do not want the header?

Reply:
Not afaik but it's more consistent with some other tools which offer the same option.